### PR TITLE
(feat) O3-1896: Number input steppers in vitals and biometrics form shouldn't exceed ranges

### PR DESCRIPTION
**Pull request by Jexsie**
… and biometrics form

## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary

- As Ian suggested, if a user is using the numeric dials, they should start at the absolute minimum and go only as far as the absolute maximum
- Updated unit test to handle the change 

## Screenshots

[Screencast from 2023-02-23 19-02-19.webm](https://user-images.githubusercontent.com/104269786/221401249-270d7523-abd2-457e-94bf-1cb3fa442754.webm)

## Related Issue

https://issues.openmrs.org/browse/O3-1896

### CODE REVIEW

**Comment by Jexsie** [comment]
cc @ibacher, @denniskigen, @gracepotma

File: packages/esm-patient-common-lib/src/useVitalsConceptMetadata.tsx [hunk 1]
```diff
@@ -17,11 +17,22 @@ export function useVitalsConceptMetadata() {
   const conceptUnits = conceptMetadata?.length
     ? new Map<string, string>(conceptMetadata.map((concept) => [concept.uuid, concept.units]))
     : new Map<string, string>([]);
+
+  const conceptRanges = conceptMetadata?.length
+    ? new Map<string, { lowAbsolute: number | string | null; highAbsolute: number | string | null }>(
```
Comment (ibacher) [inline packages/esm-patient-common-lib/src/useVitalsConceptMetadata.tsx:22]: Do we need to include the "string" type here? These should basically always be numbers. Also, note that the values may be unset (i.e., undefined). If `undefined` isn't a valid value for `min` or `max` we should probably ensure that the value will be `null`, e.g., `{ lowAbsolute: concept.lowAbsolute ?? null }`.

**Review by ibacher — COMMENTED** [review]
Thanks @Jexsie! Mostly looks good, just some TS questions.

**Comment by Jexsie** [comment]
Hey @ibacher, pushed an update for this

**Review by ibacher — APPROVED** [review]

**Pull request merged**